### DISCUSSION TITLE
VcDatasourceGeojson 的 clusterEvent 不触发问题

### DISCUSSION THREAD

**Discussion by wanfzz** [zouyaoji/vue-cesium#584] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/9a9bd876-3933-4dc3-ab84-43d2255fedb1)
在官方示例的基础测试的

**Comment by zouyaoji** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/b257a54a-e520-4f0c-b7f5-acaecaee8985)

这个漏掉没？ 建议你弄一个在线的方便看问题

**Reply by wanfzz** [reply]
可以了 感谢🙇‍
